packages/use-contractkit/src/constants.tsx

```diff
@@ -103,7 +109,7 @@ export const PROVIDERS: {
     ),
     icon: METAMASK,
     canConnect: () => isEthereumFromMetamask(),
```

Comment:
If we limit `canConnect` to `isEthereumFromMetamask`, do we need to handle the description case where it's not?